hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -506,6 +508,8 @@ private static int doBootstrap(JavaSparkContext jsc, String tableName, String ta
     cfg.schemaProviderClassName = schemaProviderClass;
     cfg.bootstrapIndexClass = bootstrapIndexClass;
     cfg.payloadClassName = payloadClassName;
+    cfg.recordMergeMode = isNullOrEmpty(recordMergeMode) ? null : RecordMergeMode.valueOf(recordMergeMode.toUpperCase());
```

Comment:
Could we add a single method in `RecordMergeMode` to handle lower and upper cases without relying on the caller to make it happen?